src/Qowaiv.Validation.DataAnnotations/AnnotatedModelValidator.cs

```diff
@@ -85,10 +85,19 @@ private void ValidateProperties(NestedValidationContext context)
     private void ValidateProperty(AnnotatedProperty property, NestedValidationContext context)
     {
         var model = context.Instance;
-        var value = property.GetValue(model);
-
         var propertyContext = context.ForProperty(property);
```

Comment:
I would define the propertyContext below the try catch because it is only used after that

Reply:
It is not. It is used to add the inaccessible error to if the `GetValue` fails.